rfc/rfc-60/rfc-60.md

```diff
@@ -196,13 +195,75 @@ for metadata table to be populated.
 
 4. If there is an error reading from Metadata table, we will not fall back listing from file system.
 
-5. In case of metadata table getting corrupted or lost, we need to have a solution here to reconstruct metadata table
-from the files which distributed using federated storage. We will likely have to implement a file system listing
-logic, that can get all the partition to files mapping by listing all the prefixes under the `Table Storage Path`.
-Following the folder structure of adding table name/partitions under the prefix will help in getting the listing and
-identifying the table/partition they belong to.
+### Integration
+This section mainly describes how storage strategy is integrated with other components and how read/write
+would look like from Hudi side with object storage layout.
+
+We propose integrating the storage strategy at the filesystem level, specifically within `HoodieWrapperFileSystem`. 
```

Comment:
We may have to introduce an abstraction that is not based on Hadoop FileSystem API. Just FYI.